### PR TITLE
Update homematicip cloud components

### PR DESCRIPTION
**Pull request by worm-ee**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14084

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by worm-ee** [comment]
Hmmm, there went something wrong. PR should just contain commit 306bb52. Do I need to cancel the PR or can I remove somehow the old commits?

File: source/_components/light.homematicip_cloud.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+sharing: true
+footer: true
+logo: homematic.png
+ha_category: Switch
+ha_release: 0.66
+ha_iot_class: "Local Push"
+---
+
+The `homematicip_cloud` light platform allows you to control
+[HomematicIP](http://www.homematicip.de) light switches through Home Assistant.
```
Comment (DubhAd) [inline source/_components/light.homematicip_cloud.markdown]: That URL redirects to http://www.homematic-ip.com

File: source/_components/sensor.homematicip_cloud.markdown [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+sharing: true
+footer: true
+logo: homematic.png
+ha_category: Sensor
+ha_release: 0.66
+ha_iot_class: "Local Push"
+---
+
+The `homematicip_cloud` sensor platform allows you to control
+[HomematicIP](http://www.homematicip.de) sensors through Home Assistant.
```
Comment (DubhAd) [inline source/_components/sensor.homematicip_cloud.markdown]: URL redirects to http://www.homematic-ip.com

File: source/_components/switch.homematicip_cloud.markdown [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+sharing: true
+footer: true
+logo: homematic.png
+ha_category: Switch
+ha_release: 0.66
+ha_iot_class: "Local Push"
+---
+
+The `homematicip_cloud` switch platform allows you to control
+[HomematicIP](http://www.homematicip.de) switches through Home Assistant.
```
Comment (DubhAd) [inline source/_components/switch.homematicip_cloud.markdown]: URL redirects to http://www.homematic-ip.com

**Review by balloob — APPROVED** [review]

**Pull request merged**